engine/src/cmd/unit_optimize_factory.h

```diff
@@ -0,0 +1,38 @@
+#ifndef UNITOPTIMIZEFACTORY_H
+#define UNITOPTIMIZEFACTORY_H
+
+/*
+ * unit_optimize_factory.cpp
+ *
+ * Copyright (C) 2020-2022 Daniel Horn, Roy Falk, Stephen G. Tuggy, and
+ * other Vega Strike contributors
+ *
+ * https://github.com/vegastrike/Vega-Strike-Engine-Source
+ *
+ * This file is part of Vega Strike.
+ *
+ * Vega Strike is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Vega Strike is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Vega Strike.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "vsfilesystem.h"
+
```

Comment:
missing the `#include <map>` - which would probably resolve the various errors

Reply:
Done and fixed.